src/resources/MachineLearning/ModelListing/ModelListingConfiguration.ts

```diff
@@ -0,0 +1,24 @@
+export interface MLListingModel {
```

Comment:
Do you mind adding jsdocs for each of those properties? You can simply copy paste what's already in swagger.

Reply:
It is added :)